central/graphql/resolvers/nodes.go

```diff
@@ -363,6 +365,58 @@ func (resolver *nodeResolver) TopVuln(ctx context.Context, args RawQuery) (Vulne
 	return &cVEResolver{root: resolver.root, data: vulns[0]}, nil
 }
 
+// TopVulnerability returns the first node vulnerability with the top CVSS score.
+func (resolver *nodeResolver) TopVulnerability(ctx context.Context, args RawQuery) (NodeVulnerabilityResolver, error) {
+	defer metrics.SetGraphQLOperationDurationTime(time.Now(), pkgMetrics.Nodes, "TopVulnerability")
+	if err := readNodes(ctx); err != nil {
+		return nil, err
+	}
+
+	if !features.PostgresDatastore.Enabled() {
+		query, err := args.AsV1QueryOrEmpty()
```

Comment:
This is a common code between `TopVulnerability` and `TopVuln`. Factor it out into common func.

Reply:
Moved the repeated block to a common func.